src/index.html

```diff
@@ -262,11 +262,190 @@ <h3 class="text-xl text-gray-50 font-semibold">
   </div>
 </section>
 
+<!-- Integrations -->
+<section id="integrations">
+  <div class="m-auto px-4 py-12 md:pt-20 flex flex-col items-center">
+    <h2 class="text-gray-900 text-2xl md:text-3xl font-semibold text-center">
+      Integrations
+    </h2>
+    <span class="mt-3 md:max-w-sm text-gray-700 font-medium text-center">
+      It's easy to get start with Livebook, whether it be on the cloud or your local machine.
+    </span>
+    <a class="mt-8 button button-outlined-gray whitespace-nowrap" href="#integrations">Browse Integrations</a>
+  </div>
+  <div class="relative">
+    <div
+    class="hidden md:block absolute right-0 h-[100%] w-[30%] bg-gradient-to-r from-transparent to-white/100"></div>
+    <div
+    class="hidden md:block absolute left-0 h-[100%] w-[30%] bg-gradient-to-l from-transparent to-white/100"></div>
+    <div class="m-auto md:flex md:flex-col justify-center items-center">
+      <div class="grid grid-cols-it-2 md:grid-cols-it-md-5 lg:grid-cols-it-lg-5 xl:grid-cols-it-xl-5 h-28 lg:h-36 xl:h-40">
+        <div class="hidden md:flex border-y border-gray-300">
+        </div>
+        <!-- Elixir -->
+        <div class="flex flex-col border-y border-x md:border-l md:border-x-0 border-gray-300 justify-center items-center">
+          <img
+          class="h-[40px] lg:h-[60px]"
+          src="/images/Integrations/Elixir_logo.svg"
+          alt="Logo"
+          />
+          <span class="text-gray-700 font-medium text-base lg:text-lg mt-2">Elixir</span>
+          <span class="text-gray-500 font-medium md:font-semibold text-xs tracking-wider sm:tracking-widest">LANGUAGE</span>
+        </div>
+        <!-- SQL -->
+        <div class="flex flex-col border-y md:border-x border-gray-300 justify-center items-center">
+          <img
+          class="h-[40px] lg:h-[60px]"
+          src="/images/Integrations/SQL_logo.png"
```

Comment:
This logo actually belongs to Microsoft Azure. My suggestion is to use one of the Remix icons for database:  https://remixicon.com/